ci/per-task-nuget-package.js

```diff
@@ -0,0 +1,164 @@
+// Overview:
```

Comment:
the ci folder is for ci scripts. we should be able to produce this locally too `node make.js package`

Reply:
...this isnt ci specific, these helpers should go in makeutil

Reply:
Awesome, didn't know that existed :) Will move the file. Had been writing it with local running in mind so should work fine.